problem_builder/templates/html/mcqblock.html

```diff
@@ -1,26 +1,29 @@
-<fieldset class="choices questionnaire">
+<fieldset class="choices questionnaire" aria-describedby="description_{{ self.html_id }}">
```

Comment:
I thought that too, but the [CAPA choicegroup template](https://github.com/edx/edx-platform/blob/432f6cdf0c6c6a6ada2549aceb072df3e9afbeb1/common/lib/capa/capa/templates/choicegroup.html#L10) do it that way.  Maybe it's because the actual description is wrapped in `<p>` tags?

Should I take it out, or leave it in?